src/pkg/packaging/snap/dotnet-runtime/snap/snapcraft.yaml

```diff
@@ -1,12 +1,21 @@
 name: dotnet-runtime
 version: 2.0.0
-summary: Cross-Platform .NET Core Runtime
+summary: Cross-Platform .NET Core Runtime. 
 description: |
-  Contains.NET Core 2.0.0 Runtime 
+  .NET Core 2.0.0 Runtime. https://dot.net/core.
 
 grade: devel
 confinement: devmode
 
+apps:
+  dotnet:
```

Comment:
apps name (dotnet) and snap name (dotnet-runtime) doesn't match . This will require adding the alias section.